### PR TITLE
Feature/467 new fields for writemarker

### LINKED ISSUE #467: New fields for WriteMarker
The WriteMarker sent to 0chain as part of a `commit_connection` transaction should contain fields for the following information
```
LookUpHash
Name
ContentHash
```
This is because Explorer needs these fields, and will read them from the WriteMarker table in the Event database.

https://github.com/0chain/0chain/issues/768
https://github.com/0chain/0chain/issues/733#issuecomment-987919888

### PR DESCRIPTION
**Pull request by princeparmar**
### Changes
added name, lookup_hash and content_hash field in write_marker.

### Tests
validated locally that lookup hash values is coming properly.

Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/blobber/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

### Associated PRs (Link as appropriate):
- 0chain:
- gosdk:
- system_test:
- zboxcli:
- zwalletcli:
- Other: ...

### CODE REVIEW

File: code/go/0chain.net/blobbercore/datastore/postgres_releases.go [hunk 1]
```diff
@@ -1,12 +1,17 @@
...
 var releases = []Migration{
-	// {
-	// 	Version:   "0.1.0",
-	// 	CreatedAt: time.Date(2021, 10, 15, 0, 0, 0, 0, time.UTC),
-	// 	Scripts: []string{
-	// 		"CREATE INDEX idx_allocation_path ON reference_objects (allocation_id,path,deleted_at);",
-	// 	},
-	// },
+	{
+		Version:   "0.1.0",
```
Comment (cnlangzi) [inline code/go/0chain.net/blobbercore/datastore/postgres_releases.go]: try to use `AutoMigrate` for write_markers table.

File: code/go/0chain.net/blobbercore/handler/object_operation_handler.go [hunk 2]
```diff
@@ -497,6 +497,7 @@ func (fsh *StorageHandler) CommitWrite(ctx context.Context, r *http.Request) (*b
 	}
 	writemarkerObj.ConnectionID = connectionObj.ConnectionID
 	writemarkerObj.ClientPublicKey = clientKey
+	writemarkerObj.WM.LookupHash = rootRef.LookupHash
```
Comment (cnlangzi) [inline code/go/0chain.net/blobbercore/handler/object_operation_handler.go]: I think we should save the lookup_hash of the current file, not root.

File: code/go/0chain.net/blobbercore/writemarker/entity.go [hunk 3]
```diff
@@ -19,6 +19,9 @@ type WriteMarker struct {
 	Timestamp              common.Timestamp `gorm:"column:timestamp" json:"timestamp"`
 	ClientID               string           `gorm:"column:client_id" json:"client_id"`
 	Signature              string           `gorm:"column:signature" json:"signature"`
+	Name                   string           `gorm:"column:name" json:"name"`
+	LookupHash             string           `gorm:"column:lookup_hash" json:"lookup_hash"`
+	ContentHash            string           `gorm:"column:content_hash" json:"content_hash"`
```
Comment (cnlangzi) [inline code/go/0chain.net/blobbercore/writemarker/entity.go]: it is unnecessary to save content_hash and name on `write_markers` table to save disk space on postgres. 
we can read them from db by lookup_hash when it is sent to blockchain.
so for `name` and `contenthash` should be `gorm:"-"`

Comment (cnlangzi) [inline code/go/0chain.net/blobbercore/writemarker/entity.go:24]: it is unnecessary to save content_hash and name on `write_markers` table to save disk space on postgres. 
we can read them from db by lookup_hash when it is sent to blockchain.

**Review by cnlangzi — CHANGES_REQUESTED** [review]
content_hash and name are missing when WM is sent to blockchain on https://github.com/0chain/blobber/blob/9e894f5f199cc66407528979bbf5c24f8994dd6b/code/go/0chain.net/blobbercore/writemarker/protocol.go#L100

**Comment by peterlimg** [comment]
@cnlangzi any updates about this PR.

**Comment by cnlangzi** [comment]
> @cnlangzi any updates about this PR.

will fix this PR soon

**Comment by cnlangzi** [comment]
A new PR will be fixed to fix #467

**Pull request closed without merging**